lib/actions.ts

```diff
@@ -86,23 +87,45 @@ export const getApplication = async (
   }
 };
 
+export const updateApplication = async ({
```

Comment:
even tho the function name is `updateApplication` it allows only to update `isEditorsPick`. we can either make it a general function that expect an `application` (excluding the id) so that it is possible to update any value. or rename the function so that the update is only for `isEditorsPick`. I prefer the 1st since it can be used in our update application page as well

Reply:
This is a really good idea!! I'll work on this!